app/assets/javascripts/external_scripts/embed_script.js.coffee

```diff
@@ -162,5 +162,14 @@ $(document).ready ->
   # Update any hidden fields with current timezone
   window.timezone ||= moment.tz.guess()
   $(".hiddenFieldTimezone").val(window.timezone)
-
-  new window.CheckEmail('#bike_owner_email')
\ No newline at end of file
+  
+  # prevent double posting
+  $('#new_bike').submit(function()) -> 
```

Comment:
Close! The direct translation from the js you posted in the issue is this:

```coffeescript
# prevent double posting
$('#new_bike').submit ->
  $this = $(this)
  if $this.data().isSubmitted
    return false
  # mark the form as processed, so we will not process it again
  $this.data().isSubmitted = true
```

The `->` is function in coffeescript. I'd recommend [js2coffee](http://js2.coffee/) to convert js, if you ever need to do something like this again.